cleverhans/utils.py

```diff
@@ -253,3 +165,17 @@ def grid_visual(data):
     # Draw the plot and return
     plt.show()
     return figure
+
+
+def conv_2d(*args, **kwargs):
+    from cleverhans.utils_keras import conv_2d
+    warnings.warn("utils.conv_2d is deprecated and may be removed on or after"
```

Comment:
I guess we can keep it for now. To be coherent, we would probably need to remove the MNIST dataset loader if we decided to remove the keras model definition.